Gemfile

```diff
@@ -2,6 +2,8 @@ source "https://rubygems.org"
 
 gemspec
 
+gem "rails", "~> #{ENV['RAILS_VERSION']}" if ENV["RAILS_VERSION"]
```

Comment:
And you should be able to use double quotes both places. Y'know for consistency :)